src/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>React & Webpack 4</title>
+</head>
+
+<body>
+    <section id="app"></section>
+</body>
```

Comment:
Add `"files.insertFinalNewline": true,` to vs code user settings file to remove the red arrow by adding a new line at the end of file.